templates/magazine/magazine.js

```diff
@@ -23,7 +24,7 @@ async function buildArticleHero(container) {
   const calendarIcon = createElement('i', { classes: ['fa', 'fa-calendar'] });
   topDetails.append(calendarIcon);
   const pubDateSpan = createElement('span', { classes: 'date' });
-  pubDateSpan.innerHTML = pubdate;
+  pubDateSpan.innerHTML = new Intl.DateTimeFormat(locale).format(new Date(pubdate)).replaceAll('-', '/');
```

Comment:
Shouldn't you be using the **getDateFromTimestamp** method here?

Reply:
no, because from article's metadata the dates come as a normal string (fe: 8/10/2024). The timestamp is generated into the `magazine-articles.xlsx` and only the blocks that read from there use the timestamp (all except the articles themselves)